### DISCUSSION TITLE
Healing doesn't seem to use all available disk resources

### DISCUSSION THREAD

**Discussion by Jazz9** [minio/minio#17685] [Q&A] [answered]
I have been testing drive replacement with minio and have found it to be going very slow.

I have applied 'mc admin config set <alias> heal max_sleep=100ms' ( and down to 10ms also )

![image](https://github.com/minio/minio/assets/39378577/a04ecdd0-539a-4590-b32e-c232858d35c1)

The cluster has 72 drives across 6 servers presented as XFS jbod. The average drive queue depths isn't exceeding 1 on any drives.

version: minio.RELEASE.2023-06-29T05-12-28Z

Two questions:

1. Are there other tunibles that I should changes?
2. Should I be changing the scanner speed also?

Glen

update - tested with heal settings: heal bitrotscan=off max_sleep=0s max_io=5000 and no improvement.

**Comment by tyler-jurgens-s2d** [comment, marked as answer]
_MINIO_HEAL_WORKERS=64 - this value defaults to number-of-cores/2

Crank up that 64 to something else, like 256/512/1024, etc. The more workers, the harder this will be on your cluster, so take smaller steps.
